bundles/org.openhab.core.semantics/model/generateTagClasses.groovy

```diff
@@ -60,6 +64,62 @@ def appendLabelsFile(FileWriter file, def line, String tagSet) {
     file.write("\n")
 }
 
+def camelToUpperCasedSnake(def tag) {
```

Comment:
Brilliant! One of my wishes (when I grow up) is to be able to do regexes like that..